charts/vault-instance/values.yaml

```
@@ -25,3 +25,6 @@ size: 1
 auth:
   namespaces:
   - secret-infra
+
+pvc:
+  storageClassName:
```

If `pvc.storageClassName` is allowed to be missing, then please provide a default empty instance of the type. For example, it the value is supposed to be a string then make this:

```suggestion
pvc:
  storageClassName: ""
```